easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -101,7 +101,10 @@ def configure_step(self):
         }
         dflags.append(comp_fam_dflags[self.toolchain.comp_family()])
 
-        libfft = os.getenv('LIBFFT')
+        if self.toolchain.options.get('openmp', False):
```

Comment:
Yeah, that's the problem, if you enable openmp in toolchainopts and hybrid using the old easyblock it wouldn't work anyway (hence your previous build error) but many people probably did not set the toolchainopt